Exercise_B/index.html

```diff
@@ -1,34 +1,49 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Flower Shop BS</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <!--If you want to use bootstrap properly this is vital (helps browser to measure the page and readjust the information accordingly-->
-
-    <!-- Link and CDN required for bootstrap -->
-
-    <!-- Google Font - Lato -->
-    <link
-      href="https://fonts.googleapis.com/css?family=Lato"
-      rel="stylesheet"
-    />
-    <!-- Custom Stylesheet -->
-    <link rel="stylesheet" href="./assets/style/stylesheet.css" />
-  </head>
-
-  <body>
-    <!--Header (flower shop logo block element)-->
-    <!--Banner Jumbotron of the image-->
-
-    <!--Four containers with img / text-->
-    <!-- 4 containers -->
-
-    <!-- Payments Div -->
-    <!-- Consultation Div -->
-
-    <!-- Delivery Div -->
-
-    <!--Scripts required for bootstrap CDN-->
-  </body>
-</html>
+<!DOCTYPE html>
+
+<head>
+    <meta charset="utf-8"> 
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/Exercise_B/assets/style/stylesheet.css">
+    <title>Flower shop</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
+</head>
+
+<body>
+    <!-- header section starts here -->
+    <header><img src='/Exercise_B/assets/images/logo.png'></header>
+    <section class='hero'>
+        <div class='overlay'>
+        </div>
+        <div class='heading'>
+            <h1>Beautiful Flowers</h1>
+            <button>Learn More</button>
+          
+        </div>
+
+    </section>
+<!-- icons section starts here -->
+    <section class='icons'>
+        <div class="container">
+            <div class="row row-cols-2">
+        <div class='icon col'>
+            <img src='/Exercise_B/assets/images/flower-icon.png'>
+            <p>Flowers for every kind of situation</p>
+        </div>
+        <div class='icon col'>
+            <img src='/Exercise_B/assets/images/payment-icon.png'>
+            <p>All major payment types accepted</p>
+        </div>
+        <div class='icon col'>
+            <img src='/Exercise_B/assets/images/phone-icon.png'>
+            <p>Free consultation</p>
+        </div>
+        <div class='icon col'>
+            <img src='/Exercise_B/assets/images/truck-icon.png'>
+            <p>City Wide delivery</p>
+            </div>
+        </div>
+        </div>
+    </section>
+</body>
+
+</html>
```

Comment:
For this exercise, the use of display flex was not used thus the icons are not flexible and also there was no background image at the back. When declaring images for use with CSS use ../ and then the href link.